src/store/sagas/auth.ts

```diff
@@ -14,12 +14,43 @@ function* signUpUser(action) {
   }
 }
 
+//http://localhost:8080/reset-password/
+function* changeUserPassword(action) {
+  try {
+    const url = location.href;
+    const [resetId] = url.split('?')[0].split('/').splice(-1);
+    const {password} = action.payload;
+
+    const data = yield call([api, 'patch'], authRequest.CHANGE_PASSWORD,  {password,resetId});
+    yield put({
+      type: authConstants.CHANGE_USER_PASSWORD_SUCCESS,
+      payload: data
+    })
+
+  } catch (errors) {
+    debugger;
+    console.log(errors, 'confirmPassword error ---------');
```

Comment:
Can we remove this?

Reply:
🐛debugger in production would have been fun  😃. Thanks for pointing that out